### PR TITLE
Feature/es 02 transform create and relate data base tables

### PR DESCRIPTION
**Pull request by Giroz22**
Establishing Relationships:
Bidirectional relationships were established among all entities based on the class diagram.

Repository Interfaces:
Repositories were created for each entity.

Database Initialization:
The schema.sql and data.sql files were created to define the database schema, including tables and relationships, and initial data.

### CODE REVIEW

File: src/main/java/com/riwi/localstorage/riwi_local_storage/domain/entities/Branch.java
Comment (SMoreno73) [inline src/main/java/com/riwi/localstorage/riwi_local_storage/domain/entities/Branch.java:1]: Remove @Data notation, and put @Setter, @Getter, @Allargument... and @NoArgument..., for this reason @toString.exclude and @hascode isn't necessary

**Review by SMoreno73 — CHANGES_REQUESTED** [review]
Remove @DaTa notation, and put https://github.com/Setter, @getter, @Allargument... and @NoArgument..., for this reason @toString.exclude and @hascode isn't necessary. Finally don't forget to put the .sql files in a folder

**Review by ralexale — APPROVED** [review]
LGTM

**Review by SMoreno73 — APPROVED** [review]
nice job!

**Pull request merged**